custom/icds_reports/reports/disha.py

```diff
@@ -64,51 +63,37 @@ def _get_columns(self):
         columns.remove("month")
         return columns
 
-    def _get_rows(self):
-        return DishaIndicatorView.objects.filter(
+    def _get_rows(self, query_master=False):
+        if query_master:
+            objects = DishaIndicatorView.objects.using('icds-ucr')
+        else:
+            objects = DishaIndicatorView.objects
+        return objects.filter(
             month=self.month,
             state_name__iexact=self.state_name
-            # batch_qs requires ordered queryset
-        ).order_by('pk').values_list(*self._get_columns())
+        ).values_list(*self._get_columns())
 
-    def _write_data_in_chunks(self, file_obj):
```

Comment:
This is just tech debt. We needed this when we had to build aww level report, now that we do block level, we don't need to paginate this heavily.